### ISSUE TITLE
Letters in team numbers are converted to numbers

### ISSUE THREAD

**Issue by wkangg** [avgupta456/statbotics#338]
When requesting a match where a team has a letter in it, the API converts the letters in the team to a number representation. An example can be found with [2023mirr](https://api.statbotics.io/v2/matches?event=2023mirr&offseason=True), 6081B, 5712B, and 7211B are represented as 608100001, 571200001 and 721100001. On [statbotics](https://www.statbotics.io/event/2023mirr#qual-matches), the letter representation is shown properly.
This is a breaking change, however, TBA does represent the teams properly and returns frc6081B, frc5712B, and frc7211B.

**Comment by avgupta456** [comment]
Yeah I only started supporting offseason teams recently, and I had already set the column to Integer instead of String. I'm rewriting a lot of it right now and it should be fixed by next season. In the meantime, you can manually convert from one representation to the other. Here's the JavaScript code the frontend uses:

```javascript
export const formatNumber = (num: number) => {
  if (num > 100000) {
    return round(num / 100000, 0).toString() + String.fromCharCode(65 + (num % 100000));
  }
  return num.toString();
};
```

https://github.com/avgupta456/statbotics/blob/master/frontend/components/utils.tsx

**Comment by avgupta456** [comment]
I ultimately decided to no longer support offseason events and offseason teams in the API. I will special case handle offseason events by giving a default EPA to offseason teams.
